.github/CODEOWNERS

```
@@ -21,4 +21,7 @@ iot/*                     @gguuss
 notebooks/*               @alixhami
 
 # Torry Yang is the primary maintainer of the Tables samples.
-/tables/               @sirtorry
+/tables/                  @sirtorry
+
+# Tianzi Cai is the primary maintainer of the Pub/Sub samples.
+pubsub/*                  @anguillanneuf
```

Torry looked into the syntax and it looks like we've been using it incorrectly 😅 
https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
```suggestion
/pubsub/                  @anguillanneuf
```

---

Glad we know better now 😉 !